infrastructure/main.tf

```diff
@@ -97,6 +98,11 @@ data "azurerm_key_vault_secret" "microservice_keys" {
   count     = "${length(local.microservice_key_names)}"
 }
 
+data "azurerm_key_vault_secret" "s2s_key" {
```

Comment:
Why is this added?

Reply:
I have no idea t by honest... it is mentioned in the read me the file so I thought to put it there 
do u want me to remove it?

Reply:
yes, you need to remove it,  the document means you need to generate secrets manually  : https://github.com/hmcts/service-auth-provider-app#generating-secret and set them to vaults in all envs
https://github.com/hmcts/service-auth-provider-app#writing-a-secret-into-all-the-needed-vaults

Reply:
ok we have generated the secrets manually - and I have removed the code

Reply:
Can u pls address this as well ?
https://github.com/hmcts/service-auth-provider-app/pull/162#pullrequestreview-267882271

Reply:
@adusumillipraveen that has been addressed now